core/src/ru/spbau/blackout/utils/InplaceSerializable.java

```diff
@@ -0,0 +1,20 @@
+package ru.spbau.blackout.utils;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+public interface InplaceSerializable {
```

Comment:
I am struggling with understanding the purpose of this interface. Java allows you to customize serialization behavior with defining your own writeObject and readObject methods. Isn't it enough?